content/en/docs/installing-on-linux/introduction/storage-configuration.md

```diff
@@ -1,37 +1,85 @@
 ---
 title: "Persistent Storage Configuration"
-keywords: 'Kubernetes, docker, KubeSphere, storage, volume, PVC'
+keywords: 'Kubernetes, docker, KubeSphere, storage, volume, PVC, KubeKey, add-on'
 description: 'Persistent Storage Configuration'
 
 linkTitle: "Persistent Storage Configuration"
 weight: 2140
 ---
+
 ## Overview
-Persistent volumes are a **Must** for KubeSphere. Therefore, before the installation of KubeSphere, a **default** 
-[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) and corresponding storage plugin should be installed on the Kubernetes cluster. As different users may choose different storage plugins, you can use [KubeKey](https://github.com/kubesphere/kubekey) to install storage plugins through 
-[add-on](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). This tutorial will introduce add-on configurations for some mainly used storage plugins.
+Persistent volume is **Must** for KubeSphere. [KubeKey](https://github.com/kubesphere/kubekey) let KubeSphere installed on different storage system by the 
+[add-on mechanism](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). General steps of KubeKey to installing KubeSphere on Linux are like:
+
+1. Install Kubernetes
+2. Install **add-on** plugin for KubeSphere
+3. Install Kubesphere by [ks-installer](https://github.com/kubesphere/ks-installer)
+
+In the config of KubeKey, ClusterConfiguration's `spec.persistence.storageClass` need to be set for ks-installer
+to create PersistentVolumeClaim ( PVC ) for KubeSphere. If empty, **default StorageClass** will be used.
+``` yaml
+apiVersion: installer.kubesphere.io/v1alpha1
+kind: ClusterConfiguration
+spec:
+  persistence:
+    storageClass: ""
+...    
+```
+
+So in Step 2, an available StorageClass **must** be installed. It includes:
```

Comment:
So in Step 2, an available StorageClass **must** be installed. ===> I don't see the reason.

Reply:
For installing KS,  ks-installer use SC to create PVC for KS workload, like `redis`, `openldap`.